jekyll/_cci2/server-3-operator-overview.adoc

```diff
@@ -47,6 +47,8 @@ Each machine reserves two vCPUs and 4GB of memory for coordinating builds. The r
 containers for running jobs. Larger machines are able to run more containers and are limited by the number of available
 cores after two are reserved for coordination.
 
+If a job's resource class requires more resources than the nomad client's instance type has available, it will remain in a pending state.  Choosing a smaller instance type for nomad clients is a way to reduce cost, but will limit the Docker resource classes CircleCI can use.  Review the https://circleci.com/docs/2.0/executor-types/#available-docker-resource-classes[available resource classes] to decide what is best for you.
```

Comment:
Should we be more explicit about the standard instance type from the terraform does not meet the requirements for our biggest supported resource class?

Reply:
I will agree with that as well. Stating the default instance and the max type it will support would be helpful.

Reply:
Good thoughts, done